CONTRIBUTING.md

```diff
@@ -26,10 +26,22 @@ Clone the repository, enter the directory, and build the provider:
 ```sh
 $ git clone git@github.com:hashicorp/terraform-provider-tfe
 $ cd terraform-provider-tfe
-$ make build
+$ make
 ```
 
-This will build the provider and put the binary in the `$GOPATH/bin` directory. To use the compiled binary, you have several different options (this list is not exhaustive):
+This will build the provider and put the binary in the project directory. To use the compiled binary, you have several different options (this list is not exhaustive):
+
+##### Using CLI config to provide a dev override (Using Terraform v0.14+)
+
+Use the rule `make devoverride` to generate a CLI config containing a dev override provider installation. This command will print a variable export that can be copied and pasted into a shell session while testing with terraform. To automatically export this override, use `eval $(make devoverride)`
```

Comment:
This is super helpful! What are your thoughts on having the script set the environment variable so all the user needs to do is invoke `make devoverride` and then go along their merry way?

Reply:
I can think of a couple reasons not to modify the running shell: I'm trying to make the fact that we are setting TF_CLI_CONFIG_FILE more visible to the user because this overwrites the default one (which is `$HOME/.terraformrc`) and I think that's pretty aggressive thing to do silently. Also, when you're running this script from a different shell, or process (like make!) it uses a different process altogether and therefore won't modify your environment.

This follows a pattern of hooking into the running shell using eval that is done by tools like tfcdev, rbenv, nodenv, direnv, etc)

Reply:
That makes a lot of sense - thank you for the explanation.